jekyll/_cci2/migrating-from-jenkins.md

```diff
@@ -46,9 +46,9 @@ Almost all configuration of CircleCI builds is stored in a file called `.circlec
 
 It’s often up to an Ops person or team to manage Jenkins servers. These people generally get involved with various CI maintenance tasks like installing dependencies and troubleshooting issues.
 
-It’s never necessary to access a CircleCI environment to install dependencies because every build starts in a fresh environment where custom dependencies must be installed automatically (ensuring that the entire build process is truly automated). Troubleshooting in the build environment can be done easily and securely by any developer using CircleCI’s [SSH feature]( {{ site.baseurl }}/2.0/ssh-access-jobs/).
+It’s never necessary to access a CircleCI environment to install dependencies because every build starts in a fresh environment where custom dependencies must be installed automatically (ensuring that the entire build process is truly automated). Troubleshooting in the execution environment can be done easily and securely by any developer using CircleCI’s [SSH feature]( {{ site.baseurl }}/2.0/ssh-access-jobs/).
```

Comment:
```suggestion
It is never necessary to access a CircleCI environment to install dependencies because every build starts in a fresh environment where custom dependencies must be installed automatically (ensuring that the entire build process is truly automated). Troubleshooting in the execution environment can be done easily and securely by any developer using CircleCI’s [SSH feature]( {{ site.baseurl }}/2.0/ssh-access-jobs/).
```